prow/jobs/istio/istio-validation.yaml

```diff
@@ -34,6 +36,8 @@ presubmits: # runs on PRs
                 cpu: 500m
     - name: pull-istio-test
       annotations:
+        description: "runs api-gateway unit tests"
```

Comment:
This is already renamed to istio instead of api-gateway. That should also be the cause of failing job.

Reply:
somehow `make jobs-definitions` did not caught this one. it should be fine now.